Features/TinyMce/functions.php

```diff
@@ -53,5 +20,50 @@
 add_filter('tiny_mce_before_init', function ($init) {
     // Add block format elements you want to show in dropdown
     $init['block_formats'] = 'Paragraph=p;Heading 1=h1;Heading 2=h2;Heading 3=h3;Heading 4=h4;Heading 5=h5;Heading 6=h6';
+    
+    // Load Styleformat Dropdown and parse it into TinyMCE
+    $options = Feature::getOptions('flynt-tiny-mce');
+    if (isset($options[0]) && isset($options[0]['styleformatsConfigPath'])) {
+        $configPath = $options[0]['styleformatsConfigPath'];
```

Comment:
Do you think this feature could get more complex in the future? If we have more parameters passed, it might be worth considering nesting the config paths in another level.